.kokoro/tests/run_tests.sh

```diff
@@ -61,6 +61,8 @@ if [[ "$SCRIPT_DEBUG" != "true" ]]; then
     export LOCATION_ID=us-east1
     # For Datalabeling samples to hit the testing endpoint
     export DATALABELING_ENDPOINT="test-datalabeling.sandbox.googleapis.com:443"
+    # For Transcoder samples
+    export GOOGLE_CLOUD_PROJECT_NUMBER="779844219229"
```

Comment:
Ideally this would be provided by kokoro.  (ie ask devrel to provide it).  Also, does your api support Project Names?  if so, that is preferred.

Reply:
Confirmed that this is the project number for the samples test project.

Reply:
The API supports project numbers and project IDs.

A project name would be "projects/<project_number or project_id>", so I'm guessing you were referring to project IDs.
https://google.aip.dev/cloud/2510#project-identifier-format

Reply:
There are a few api's that only allow numbers, so I like to know what they are before approving them.  Since GOOGLE_CLOUD_PROJECT is already set in the environment for you, do you really need this?

Reply:
Transcoder API support both project numbers and IDs.